researchstack-sdk/src/main/java/org/sagebionetworks/bridge/researchstack/BridgeDataProvider.java

```diff
@@ -16,23 +18,26 @@
 import org.researchstack.backbone.utils.LogExt;
 import org.researchstack.backbone.utils.ObservableUtils;
 import org.researchstack.skin.AppPrefs;
-import org.researchstack.skin.DataProvider;
-import org.researchstack.skin.DataResponse;
-import org.researchstack.skin.ResourceManager;
-import org.researchstack.skin.model.SchedulesAndTasksModel;
+import org.researchstack.backbone.DataProvider;
```

Comment:
Do we have a high-level strategy or plan for how we want to split things between backbone, skin, and Bridge SDK?

I feel a more generic version of DataProvider could go in backbone, the current DataProvider seems pretty specific to Bridge... Or we can see how things evolve and clean up before merging into master.